sql/moz-fx-data-shared-prod/telemetry_derived/clients_scalar_probe_counts_v1/query.sql

```diff
@@ -122,20 +122,51 @@ user_aggregates AS (
     app_version,
     app_build_id,
     channel),
-
+build_ids AS (
+  SELECT
+    app_build_id,
+    channel,
+  FROM
+    user_aggregates
+  GROUP BY
+    1,
+    2
+  HAVING
+    CASE
+    WHEN
+      channel = 'release'
+    THEN
+      SUM(user_count) > 625000
+    WHEN
+      channel = 'beta'
+    THEN
+      SUM(user_count) > 9000
+    WHEN
+      channel = 'nightly'
+    THEN
+      SUM(user_count) > 375
+    ELSE
+      SUM(user_count) > 100
+    END
+),
 bucketed_booleans AS (
   SELECT
     client_id,
     os,
     app_version,
-    app_build_id,
+    user_aggregates.app_build_id,
     channel,
     user_count,
-    os = 'Windows' and channel = 'release' AS sampled,
+    os = 'Windows'
+    AND channel = 'release' AS sampled,
     udf_boolean_buckets(scalar_aggregates) AS scalar_aggregates
   FROM
-    user_aggregates),
-
+    user_aggregates
+  INNER JOIN
```

Comment:
I think we don't need the inner join here because we have it below right?

Reply:
Good catch. Will remove it